include/seastar/util/closeable.hh

```diff
@@ -122,6 +122,11 @@ public:
     /// Construct an object that will auto-stop \c obj when destroyed.
     /// \tparam obj the object to auto-stop.
     deferred_stop(Object& obj) noexcept : _obj(obj) {}
+    /// Moves the \c deferred_stop into a new one, and
+    /// the old one is canceled.
+    deferred_stop(deferred_stop&& x) noexcept : _obj(x._obj), _stopped(x._stopped) {
```

Comment:
nit: I believe it can be a bit simpler:

```c++
deferred_stop(deferred_stop&& x) noexcept : _obj(x._obj), _stopped(std::exchange(x._stopped, true)) {}
```

Reply:
yeah. thought it was too long. but i agree it's simpler. changed as suggested.